test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Link.Encoded.html

```diff
@@ -39,63 +39,63 @@
 
     <!-- Fallback to static href -->
     <link href="HtmlEncode[[/styles/site.min.css?a=b&c=d]]" rel="stylesheet" data-extra="test" title='"the" title' />
```

Comment:
Suggest updating the `.cshtml` file to add more attribute variation. Looks like this is the only updated case that adds anything beyond `rel` and `data-extra`. Would like to see at least one w/ dynamic content in the attribute value and one with only `rel` and `asp-*` attributes.